index.bs

```diff
@@ -111,32 +172,71 @@ dictionary MediaStreamTrackProcessorInit {
 1. If |init|.{{MediaStreamTrackProcessorInit/track}} is not a valid {{MediaStreamTrack}},
     throw a {{TypeError}}.
 2. Let |processor| be a new {{MediaStreamTrackProcessor}} object.
-3. Assign |init|.{{MediaStreamTrackProcessorInit/track}} to the `[[track]]`
-    internal slot of |processor|.
-3. Assign |init|.{{MediaStreamTrackProcessorInit/maxBufferSize}} to the `[[maxBufferSize]]`
-    internal slot of |processor|.
-4. Return |processor|.
+3. Assign |init|.{{MediaStreamTrackProcessorInit/track}} to |processor|.`[[track]]`.
+4. Assign |init|.{{MediaStreamTrackProcessorInit/maxBufferSize}} to |processor|.`[[maxBufferSize]]`.
+5. Set the `[[queue]]` internal slot of |processor| to an empty {{Queue}}.
+6. Set |processor|.`[[numPendingReads]]` to 0.
+7. Set |processor|.`[[isClosed]]` to false.
+8. Return |processor|.
+
+<p class="note">
+The application may detect that frames have been dropped by noticing that there is a gap in the
+timestamps of the frames.
+</p>
```

Comment:
This is duplicated on lines 231-234. It seems out-of-place here.

Reply:
Removed